src/test/ui/coherence/deep-bad-copy-reason.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0204]: the trait `Copy` may not be implemented for this type
+  --> $DIR/deep-bad-copy-reason.rs:33:15
+   |
+LL | pub struct List<'tcx, T>(Interned<'tcx, ListS<T>>);
+   |                          ------------------------ this field does not implement `Copy`
+...
+LL | impl<'tcx, T> Copy for List<'tcx, T> {}
+   |               ^^^^
+   |
+note: the `Copy` impl for `Interned<'tcx, ListS<T>>` requires that `OpaqueListContents: Sized`
```

Comment:
I think it could be confusing here that we refer to `OpaqueListContents` but it isn't present in the type we're pointing at (particularly for a beginner) - it makes sense why we are doing so, and I think the additional note here is helpful, but maybe we could reword this slightly to emphasise that the type _contains_ `OpaqueListContents` which is preventing `Copy`. 

If `ListS<T>` was from another crate, would this still print the additional note? If so, it might make sense to skip the note in that case as it wouldn't be actionable for the user.

Reply:
> If `ListS<T>` was from another crate, would this still print the additional note?

I believe it would, but I still think it's useful to note this because it still shows the user _why_. Anywho, I left a FIXME because I don't really know the best way to make this error more descriptive.